composer.json

```diff
@@ -18,7 +18,7 @@
     "provide": {
         "php-http/async-client-implementation": "*",
         "php-http/client-implementation": "*",
-        "psr/cache-implementation": "1.0|2.0",
+        "psr/cache-implementation": "1.0|2.0|3.0",
```

Comment:
Sorry for asking instead of testing myself. 

Are we sure we still support 1.0?

Reply:
@Nyholm Yes, we are. As long as we don't add the parameter types that v2 introduced.